packages/components/Separator/src/Separator.types.ts

```diff
@@ -1,11 +1,26 @@
 import { ViewProps, ColorValue } from 'react-native';
 
-interface SeparatorPropTokens {
+export const separatorName = 'Separator';
+
+export const SeparatorInsetSpacings = [0, 16, 56, 68, 72, 108] as const;
+
+export type SeparatorInsetSpacing = typeof SeparatorInsetSpacings[number];
+
+export interface SeparatorPropTokens {
   /**
    * Specifies whether the separator is vertical or horizontal. If true, then it's vertical.
    * @defaultValue 'false'
    */
   vertical?: boolean;
+
+  /**
+   * Inset Spacing in pixels.
+   *
+   * Inset Spacing is restricted to a limited set of supported values recommended for most uses (see `SeparatorInsetSpacing`) and
+   * based on design guidelines for the Separator control.
+   * @defaultvalue 0
```

Comment:
Please mark this prop as mobile only

Reply:
Added.

Reply:
nit: I've seen `@platform` tags for these sorts of things before.

Reply:
Found  @platform tags , thanks I didn't knew it :)

Reply:
Added.

Reply:
I _think_ it might be better if the platforms are space delineated rather than comma delineated? It would be easy to check by just hovering over this prop in VS Code. Normally the platform tag only has one platform, so I haven't figured out the best way to list multiple platforms. I went with spaces earlier, would like to know if something else is better.

Reply:
Added space delineated for now.